src/focus-ring.js

```diff
@@ -29,22 +31,31 @@ document.addEventListener('DOMContentLoaded', function() {
 	    }
 	    return triggers;
 	},
+        _addClass = function(el, htmlClass) {
+            el.className += ' '+htmlClass;   
+        },
+        _removeClass = function(el, htmlClass) {
```

Comment:
Good call! I've snagged the regex helper function from the classList Polyfill and followed their lead. Feels cleaner :) (And because I'll never understand Regex :) ha!)